source/common/http/alternate_protocols_cache.h

```diff
@@ -17,12 +17,12 @@ namespace Http {
 // See https://tools.ietf.org/html/rfc7838 for HTTP Alternate Services and
 // https://datatracker.ietf.org/doc/html/draft-ietf-dnsop-svcb-https-04 for the
 // "HTTPS" DNS resource record.
-class AlternateProtocols {
+class AlternateProtocolsCache {
 public:
   // Represents an HTTP origin to be connected too.
   struct Origin {
   public:
-    Origin(absl::string_view scheme, absl::string_view hostname, int port);
+    Origin(absl::string_view scheme, absl::string_view hostname, uint32_t port);
```

Comment:
Would `uint16_t` be more appropriate for port numbers?

Reply:
Matching Envoy makes sense. I'm surprised that it uses uint32_t, but let's not shave that yak today.